### PR TITLE
Fix outdated links

### PR DESCRIPTION
**Pull request by vanillyn**

### CODE REVIEW

**Comment by kekkodance** [comment]
instead of removing the megathread, can you add a notice saying it's discontinued?

**Comment by vanillyn** [comment]
Oh, alright!

**Comment by kekkodance** [comment]
@Helmasko all good?

**Comment by vanillyn** [comment]
is there a libretro section?

**Comment by vanillyn** [comment]
thats all i can find, but i may add an emulation tools page along with libretro on another pr

File: Emulation/Emulators.md [hunk 1]
```diff
@@ -35,7 +36,7 @@ description: Comprehensive list of Emulators.
 
 | Platform | Emulator |
 | --- | --- |
-|Windows|[EMU7800](http://emu7800.sourceforge.net/)|
```
Comment (Helmasko) [inline Emulation/Emulators.md:38]: new emu7800 link
https://github.com/emu7800/emu7800.github.io
source: https://emulation.gametechwiki.com/index.php/Atari_7800_emulators

File: Literature/e-Books.md [hunk 2]
```diff
@@ -58,7 +58,6 @@ description: Reliable sources to obtain e-Books.
 - [**Bypass Paywalls Clean**](https://gitlab.com/magnolia1234/bypass-paywalls-chrome-clean) - Bypass News Article paywalls.  
 *<small>A more actively maintained fork (with no Google analytics) of [Bypass Paywalls](https://github.com/iamadamdev/bypass-paywalls-chrome).</small>*
 - [**Unpaywall**](https://unpaywall.org/) Browser Extension: [Chromium](https://chrome.google.com/webstore/detail/unpaywall/iplffkdpngmdjhlpjmppncnlhomiipha) & [Firefox](https://addons.mozilla.org/en-GB/firefox/addon/unpaywall/) - When you view a research article, Unpaywall automatically looks for a copy in its index. If it finds one, click the green tab to read the article.
-- [**Outline**](https://outline.com/) - Removes clutter from News Articles.
```
Comment (Helmasko) [inline Literature/e-Books.md:61]: Im planning https://12ft.io/ could be a replacement for this, anyone have any experience with it?

Comment (kekkodance) [reply Literature/e-Books.md:61]: yeh its great

**Comment by Helmasko** [comment]
I wont touch the Emulators section further since were planning on completely revamping it, so I'll merge this for now and work on the Emulators section with the other collaborators. Thanks for the contribution!

**Pull request merged**